### PR TITLE
docs: docstring sets

### PR DESCRIPTION
**Pull request by theUnkownName**

### CODE REVIEW

File: jina/types/sets/chunk.py [hunk 1]
```diff
@@ -5,6 +5,16 @@
 
 
 class ChunkSet(DocumentSet):
+    """
+    :class:`ChunkSet` inherits from :class:`DocumentSet`.
+    It's a subset of Documents.
+
+    :param docs_proto:
+    :type docs_proto: :class:`Document`
+    :para reference_doc:
```
Comment (bwanglzu) [inline jina/types/sets/chunk.py:14]: param

File: jina/types/querylang/__init__.py [hunk 2]
```diff
@@ -96,10 +99,19 @@ def priority(self, value: int):
 
     @property
     def name(self) -> str:
-        """Get the name of the driver that the query language attached to """
+        """
+        Get the name of the driver that the query language attached to
+        
+        :rtype: str
```
Comment (bwanglzu) [inline jina/types/querylang/__init__.py:105]: it's a bit wired to see rtype without return

**Review by bwanglzu — CHANGES_REQUESTED** [review]

File: jina/types/querylang/__init__.py (hunk 2 above)
Comment (theUnkownName) [reply jina/types/querylang/__init__.py:105]: I was going to add it, but thanks for the reminder

File: jina/types/sets/querylang.py [hunk 3]
```diff
@@ -20,17 +20,23 @@
...
     """
 
     def __init__(self, querylang_protos: 'RepeatedContainer'):
+        """Set constructor method."""
         super().__init__()
         self._querylangs_proto = querylang_protos
         self._querylangs_map = {}
 
     def insert(self, index: int, ql: 'QueryLang') -> None:
+        """Insert :param:`ql` into `_querylangs_proto`."""
```
Comment (bwanglzu) [inline jina/types/sets/querylang.py:39]: insert xxx at :param:`index`

**Review by bwanglzu — CHANGES_REQUESTED** [review]
Looks great! Only minor things

File: jina/types/sets/querylang.py (hunk 3 above)
Comment (theUnkownName) [reply jina/types/sets/querylang.py]: Good catch!

**Review by nan-wang — DISMISSED** [review]
LGTM👍

File: jina/types/sets/match.py [hunk 4]
```diff
@@ -5,14 +5,27 @@
...
 class MatchSet(DocumentSet):
+    """
+       :class:`MatchSet` inherits from :class:`DocumentSet`.
+       It's a subset of Documents that represents the matches
+
+       :param docs_proto: Set of matches of the `reference_doc`
+       :type docs_proto: :class:`Document`
+       :para reference_doc: Reference :class:`Document` for the sub-documents
+       :type reference_doc: :class:`Document`
+       """
```
Comment (bwanglzu) [inline jina/types/sets/match.py:16]: the indentation seems to be not aligned

**Review by bwanglzu — COMMENTED** [review]
The rest looks good to me!

File: jina/types/sets/match.py (hunk 4 above)
Comment (theUnkownName) [reply jina/types/sets/match.py]: You're right

**Review by bwanglzu — APPROVED** [review]
LGTM!

**Pull request merged**